banners/desktop/C24_WMDE_Desktop_DE_09/banner_var.ts

```diff
@@ -1,76 +0,0 @@
-import { createVueApp } from '@src/createVueApp';
```

Comment:
We usually don't delete the entry points in 2nd commit because if we have to modify them in the 3rd commit it makes it hard to review what changed in there.

Reply:
Yes, I deleted it by mistake. In my previous banners I did it as you explained. Sorry, I will be careful next time.